docs/modules/i3s/api-reference/i3s-loader.md

```diff
@@ -178,7 +178,7 @@ const viewport = new WebMercatorViewport({latitude, longitude, zoom, ...})
 tileset3d.update(viewport);
 
 // Viewport changes (pan zoom etc)
-tileset3d.update(viewport);
+tileset3d.selectTiles(viewport);
```

Comment:
I am starting to feel it could be a good time to audit the API naming here. Neither of these function names seem natural to me in this context. 

I.e. what kind of `selection` or `update` are we talking about here?

Reply:
Yeah I think maybe `load` is the most intuitive from a user's perspective. Like, "given this viewport, please load the tiles needed to render it".